include/mme-app/ue_table.h

```diff
@@ -66,6 +66,7 @@ enum ue_stages{
   DETACH_STAGE2_DS_DONE,
   DETACH_STAGE2,
   DETACH_DONE,
+  CTX_RELEASE_STAGE,
```

Comment:
This s critical.  Lets add enough information about what are these states mean.

Reply:
done.